deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,382 @@
+        'session': args.session
+    }
+
+    return artillery_params
+
+
+def add_options_to_artillery(config: str, options: dict) -> None:
+    """Add missing parameters to Artillery config.
+
+    Args:
+        config (str): YAML de Artillery.
+        options (dict): Parametros de Artillery.
+    """
+    # Open YML
+    with open(config) as f:
+        data = yaml.safe_load(f)
+
+    # Check `config` in Artillery YAML
+    if 'config' not in data.keys():
+        data['config'] = {}
+
+    # Check `variables` in Artillery YAML
+    if 'variables' not in data['config'].keys():
+        data['config']['variables'] = {}
+
+    # Add Options to Artillery
+    for key in options:
+        if key not in data['config']['variables'].keys():
+            data['config']['variables'][key] = f'{{{{ { key } }}}}'
+
+    # Save YML
+    with open(config, "w") as f:
+        yaml.dump(data, f)
+
+
+def process_script_arguments(args: argparse) -> None:
+    """Process script arguments, create folders and generate necessary files.
+
+    Args:
+        args (argparse): Script parameters.
+    """
+    # Format .auth Path
+    if not isabs(args.session):
+        args.session = f'{getcwd()}/{args.session}'
+
+    # Check Artillery Result Types
+    check_artillery_result_types(args.type)
+
+    # Provide Necessary Directories
+    provide_directories(args)
+
+    # Create Session File
+    create_session_file(args.session, args.user)
+
+    # Check Artillery Parameters
+    add_options_to_artillery(args.artillery, gen_artillery_params(args))
+
+
+def gen_log_filename(log_path: str) -> str:
+    """Generate log file name.
+
+    Args:
+        log_path (str): Path of the logs.
+
+    Returns:
+        str: File name of the log (include path).
+    """
+    return log_path + datetime.now().strftime(f"log-%Y%m%d%H%M%S.log")
+
+
+def gen_url(ip: str) -> str:
+    """Generate dashboard url.
+
+    Args:
+        ip (str): IP of the dashboard.
+
+    Returns:
+        str: Complete url of the dashboard.
+    """
+    url_format = 'https://'
+
+    return f'{url_format}{ip}'
+
+
+def gen_csv_filename(csv_path: str, type: str) -> str:
+    """Generate csv file name (per type).
+
+    Args:
+        csv_path (str): Path of the CSVs.
+        type (str): CSV data type.
+
+    Returns:
+        str: File name of the csv (include type and path).
+    """
+    return csv_path + datetime.now().strftime(f"{type}-%Y%m%d%H%M%S.csv")
+
+
+def convert_json_to_csv(args: argparse, json_output: str) -> None:
+    """Convert data from JSON format to CSV format.
+
+    Args:
+        args (argparse): Script parameters.
+        json_output (str): Path and file name of the log.
+    """
+    for type in args.type:
+        with open(json_output) as f:
+            data = json.load(f)
+
+        csv_filename = gen_csv_filename(args.csv, type)
+
+        df = pd.json_normalize(data[type])
+        df.to_csv(csv_filename, index=False)
+
+
+def run_artillery(args: argparse) -> None:
```

Comment:
Same here, use Namespace

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/a07ebb3ed73c7bc3ab8fce58bf98032b832575f8